### DISCUSSION TITLE
Sugestão de onde adquirir certificado digital

### DISCUSSION THREAD

**Discussion by megurost** [DyegoCosta/trabalhando-remoto#159] [Q&A]
Pessoal, onde vocês compram certificados e-CNPJ?
Vou migrar pra Contabilizei e precisava de um certificado A1. Pesquisando vi que os preços variam bastante, mas não sei até onde os sites são confiáveis

**Comment by igorlvicente** [comment]
O de CPF eu fiz pessoalmente (necessário pra abrir a empresa).
O de CNPJ eu fiz online com a Ello's. Não teve nenhuma razão específica além de poder fazer remoto por uma vídeo chamada.

**Comment by oidoug** [comment]
A contabilizei tem uma parceria com a Soluti:

contabilizei.com.br/certificado-digital

Eu uso, por causa da parceria a importação fica transparente. Eu não gostei muito do atendimento deles, e a primeira vez tive que ir num ponto físico fazer o cadastro. Acho que agora é todo remoto também.

**Comment by megurost** [comment]
Opa, valeu galera
Vou dar uma olhada nesses aí
